Fabric.Identity.API/scripts/Register-Identity-IdPSS.ps1

```diff
@@ -37,9 +39,15 @@ if($null -ne $tenants) {
             -tenantId $tenant `
             -clientSecret $clientSecret `
             -clientId $clientId `
-            -installConfigPath $installConfigPath
+            -installConfigPath $installConfigPath `
+			-appName $appName
```

Comment:
$appName doesn't look like it's being set

Reply:
I removed the appName variable by accident in the merge. Good catch. I will add it back.